app/templates/administrative-units/administrative-unit/core-data/edit.hbs

```diff
@@ -4,7 +4,25 @@
       <:title>Bewerk kerngegevens</:title>
       <:subtitle>{{@model.administrativeUnit.name}}</:subtitle>
       <:action>
-        <ReportWrongData />
+        <div>
```

Comment:
This looks fine but because the buttons aren't nested in the form anymore they aren't linked and clicking the button won't submit the form.

Adding an id to the form and using the [form attribute on the button](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-form) should work and it's probably needed on all the other pages as well.